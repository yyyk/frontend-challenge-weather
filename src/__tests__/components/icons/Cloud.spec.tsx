import { render } from '@testing-library/react';
import Cloud from 'src/components/icons/Cloud';

describe('<Cloud>', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Cloud />);
    expect(asFragment()).toMatchSnapshot();
  });
});
