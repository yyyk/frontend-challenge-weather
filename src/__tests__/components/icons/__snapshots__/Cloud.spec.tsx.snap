// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cloud> matches snapshot 1`] = `
<DocumentFragment>
  <svg
    aria-labelledby="title"
    fill="none"
    height="24"
    role="img"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <title
      id="title"
    >
      Cloudy
    </title>
    <clippath
      id="a"
    >
      <path
        d="m0 0h24v24h-24z"
      />
    </clippath>
    <g
      clip-path="url(#a)"
    >
      <path
        d="m9.75 21.75c-.674 0-1.346-.153-1.949-.442-2.235-1.072-3.182-3.761-2.11-5.997.743-1.55 2.337-2.552 4.059-2.552.37 0 .736.045 1.093.136.902-2.754 3.486-4.646 6.414-4.646 1.811.002 3.505.706 4.779 1.983 1.273 1.277 1.974 2.972 1.971 4.775-.004 3.718-3.032 6.742-6.75 6.742zm7.518-1.5c2.88 0 5.236-2.353 5.239-5.244.002-1.402-.543-2.721-1.533-3.714s-2.308-1.54-3.711-1.542c-2.524 0-4.693 1.795-5.164 4.268 0 0-.003.016-.005.022-.061.305-.094.634-.095.961 0 .214.014.433.04.647.005.04.007.071.007.102 0 .377-.282.697-.656.744-.032.004-.063.006-.094.006-.377 0-.696-.282-.744-.656-.034-.279-.052-.563-.052-.844.001-.211.012-.422.033-.631-.259-.072-.521-.109-.783-.109-1.149 0-2.211.667-2.706 1.7-.714 1.489-.083 3.281 1.406 3.996.381.183.833.284 1.308.294h7.495z"
        fill="#999"
      />
      <path
        d="m4.5 13.734c-.255 0-.489-.127-.628-.34-.423-.647-.697-1.382-.803-2.144h-2.319c-.414 0-.75-.336-.75-.75s.336-.75.75-.75h2.327c.015-.107.032-.207.053-.305.157-.75.479-1.459.936-2.069l-1.649-1.649c-.142-.141-.22-.33-.22-.53s.078-.389.22-.53c.141-.142.33-.22.53-.22s.389.078.53.22l1.651 1.651c.092-.07.18-.132.268-.19.637-.416 1.358-.689 2.104-.8v-2.328c0-.414.336-.75.75-.75s.75.336.75.75v2.327c.122.016.235.035.345.058.74.155 1.44.47 2.044.916l1.634-1.634c.142-.142.33-.22.53-.22s.389.078.53.22c.142.141.22.33.22.53s-.078.389-.22.53l-1.63 1.63c.078.1.146.197.21.294.046.071.091.143.134.217.101.173.129.375.078.569s-.174.356-.347.457c-.114.067-.245.103-.378.103-.266 0-.514-.142-.648-.371-.031-.052-.062-.104-.096-.155-.548-.839-1.389-1.414-2.37-1.619-.256-.053-.517-.08-.777-.08-.724 0-1.43.211-2.043.612-.839.548-1.414 1.389-1.619 2.37-.206.98-.017 1.982.531 2.821.109.168.147.368.106.564s-.156.364-.324.474c-.122.079-.264.121-.41.121z"
        fill="#ffc700"
      />
    </g>
  </svg>
</DocumentFragment>
`;
